README.md

```
@@ -23,6 +23,7 @@
     *   [Testing Methods with Callbacks](#testing-methods-with-callbacks)
     *   [Testing Closure-Based Dependencies](#testing-closure-based-dependencies)
     *   [Test Isolation for Concurrent Testing](#test-isolation-for-concurrent-testing)
+    *   [Test-Scoped Default Values](#test-scoped-default-values)
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)